tests/format/jsx/text-wrap/__snapshots__/format.test.js.snap

```diff
@@ -1,5 +1,72 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`issue-16897.js format 1`] = `
+====================================options=====================================
+parsers: ["flow", "typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+function HelloWorld(   ) {
+  return (
+    <div>
+      <div>
+        foo
+        <br />
+        bar{' '}
+        <span className="font-semibold">
+          foobar foobar foobar foobar 12345
+        </span>, foobar foobar foobar
+      </div>
+    </div>
+  )
+}
+
+function HelloWorld() {
```

Comment:
We don't need add the output here, the tests will format output to test stability.

Reply:
Thank you for your review. Let me get sure what you point. Do you mean idempotency is checked by FULL_TEST=1 so second HelloWorld() is not necessary, and first one is still necessary?

Reply:
Thank you for reaction. I removed the latter one.